e2e/test/scenarios/embedding/sdk-iframe-embedding/sdk-iframe-embedding.cy.spec.ts

```diff
@@ -317,10 +317,10 @@ H.describeWithSnowplowEE(
       H.resetSnowplow();
       H.prepareSdkIframeEmbedTest({ signOut: false });
       H.enableTracking();
-      cy.signOut();
```

Comment:
since we need to test the embedded analytics JS wizard we need to keep our login.